src/qibo/result.py

```diff
@@ -237,6 +246,37 @@ def frequencies(self, binary=True, registers=False):
 
         return self._frequencies
 
+    def probabilities(self, qubits=None):
+        """Calculate the probabilities as frequencies / nshots
+
+        Returns:
+            The array containing the probabilities of the measured qubits.
+        """
+        nqubits = len(self.measurement_gate.qubits)
+        if qubits is None:
+            qubits = range(nqubits)
+        else:
+            if not set(qubits).issubset(self.measurement_gate.qubits):
+                raise RuntimeError(
+                    "Asking probabilities for qubits that were not measured."
+                )
+            else:
```

Comment:
```suggestion
                raise_error(RuntimeError, f"Probabilities were asked for qubits {qubits}, but they were not measured.")
```

Reply:
It's best to delegating the raising of errors to the `qibo`'s `raise_error` function in order to keep the code unified throughout. Then, this should be changed everywhere else.